spec/observables/combineLatest-spec.js

```diff
@@ -3,15 +3,69 @@ var Rx = require('../../dist/cjs/Rx');
 var Observable = Rx.Observable;
 
 describe('Observable.combineLatest', function () {
+  function concat(x, y) { return '' + x + y; }
+
+  it('should be never when sources are never and never', function () {
```

Comment:
Isn't this case already exists in [spec/operators/combineLatest-spec](https://github.com/ReactiveX/RxJS/blob/master/spec/operators/combineLatest-spec.js#L77)?